Northstar.CustomServers/mod/scripts/vscripts/weapons/_vortex.nut

```diff
@@ -78,12 +79,20 @@ global const VORTEX_REFIRE_ROCKET				= "rocket"
 global const VORTEX_REFIRE_GRENADE				= "grenade"
 global const VORTEX_REFIRE_GRENADE_LONG_FUSE	= "grenade_long_fuse"
 
-const VortexIgnoreClassnames = {
+table<string, bool> VortexIgnoreClassnames = {
 	["mp_titancore_flame_wave"] = true,
 	["mp_ability_grapple"] = true,
 	["mp_ability_shifter"] = true,
 }
-
+bool function RegisterNewVortexIgnoreClassname(string classname, bool shouldignore)
```

Comment:
You should use this implementation otherwise it won't be possible to change the value after the weapon gets registered.

```suggestion
void function RegisterNewVortexIgnoreClassname(string classname, bool shouldignore)
{
	VortexIgnoreClassnames[classname] <- shouldignore
}
```

Reply:
Thats a good point though unless I'm misreading (not great at GitHub tbh) that wouldn't allow you to either, <- only allows you to create new values in the table not edit existing ones, meaning attempting to edit existing entries would cause issues. will implement that functionality